### DISCUSSION TITLE
AI在視覺上的應用

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#624] [科技新知]
https://www.moea.gov.tw/MNS/doit/industrytech/IndustryTech.aspx?menu_id=13545&it_id=422
疫情的嚴峻，使得很多事情演化成要遠距來操作，而AI在這塊也能幫得上忙!利用AI鏡頭能追蹤移動中的使用者，讓設備能隨時帶著走。

**Comment by Rosancl** [comment]
知道AI有很多面向的應用，但是是第一次看關於影像感測AI技術類的文章，如文中提到的智慧化、5G的發展，並且加上現今疫情的影像，相信之後具AI功能的視覺感測器會成為趨勢之一。
